cmd/main.go

```diff
@@ -345,7 +357,11 @@ func main() { //nolint:funlen,maintidx,gocyclo
 		os.Exit(1)
 	}
 
-	webhook.Init(mgr)
```

Comment:
In the webhook file, there were a `/go:build !nowebhook` build directive to avoid registering webhooks for local execution, I don't see a similar logic being implemented in the `register.go` methods

Reply:
Good catch, let me integrate it.